src/components/TodoMain/TodoMain.tsx

```diff
@@ -0,0 +1,61 @@
+import cn from 'classnames';
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+export const TodoMain: React.FC<{ todos: Todo[] | null }> = ({ todos }) => {
+  const [changeCheck, setChangeCheck] = useState<number>(-1);
+
+  const onSubmitChanges = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setChangeCheck(-1);
+  };
+
+  return (
+    <section className="todoapp__main">
+      {todos?.map((todo) => {
+        const { completed, title, id } = todo;
+
+        return (
+          <div className={cn('todo', { completed })} key={id}>
```

Comment:
`className={cn('todo', { completed })}` - how is it expected to behave?

Reply:
Thank you Nazar, I'll wait for approval and then I'll update them in the next project too... 😅☕️